src/backend/storage/file/buffile.c

```diff
@@ -1033,13 +1033,13 @@ BufFilePledgeSequential(BufFile *buffile)
 
 #define BUFFILE_ZSTD_COMPRESSION_LEVEL 1
 
-void *
+static void *
```

Comment:
Although its not strictly required to have a function prototype in C, some compilers do require them, regardless of being static. 
Even this file has examples of function prototypes for static functions - look at makeBufFile().
I suggest to add prototypes in the beginning of this file and protect them with a HAVE_LIBZSTD macro.

Reply:
Thanks for the notice, done